packages/frontend/src/server/middlewares/ErrorHandler.ts

```diff
@@ -1,19 +1,15 @@
 import type { Logger } from '@l2beat/backend-tools'
 import type { NextFunction, Request, Response } from 'express'
 
-export function ErrorHandler(
-  err: Error,
-  req: Request,
-  res: Response,
-  _: NextFunction,
-  appLogger: Logger,
-) {
-  res.status(500)
-  appLogger.error('Error processing request', {
-    error: err instanceof Error ? err.message : String(err),
-    method: req.method,
-    url: req.originalUrl,
-    status: res.statusCode,
-  })
-  res.send('Internal Server Error')
+export function ErrorHandler(appLogger: Logger) {
+  return (err: Error, req: Request, res: Response, _: NextFunction) => {
+    res.status(500)
+    appLogger.error('Error processing request', {
+      error: err instanceof Error ? err.message : String(err),
```

Comment:
generate a uuid and add it to the log and send the UUID in the response so that when you get a 500 you know which log to search for